tern/analyze/docker/dockerfile.py

```diff
@@ -363,3 +364,41 @@ def split_multistage_dockerfile(dfobj):
         # dfobj_temp not empty, append to the list
         dfobj_list.append(copy.copy(dfobj_temp))
     return dfobj_list
+
+
+def get_multistage_image_dockerfiles(dfobj_multi):
+    """Given a multistage dockerfile object, return a list of structures
+     for building image."""
+    file_path_list = []
+    structure = []
+    idx = 0
+    for command_dict in dfobj_multi.structure:
+        if command_dict['instruction'] == 'FROM':
+            if structure:
+                file_path = dfobj_multi.filepath + '_%d' % (idx)
+                # we make a new dir for the dockerfile of each stage.
+                if not os.path.isdir(file_path):
+                    os.mkdir(file_path)
```

Comment:
Since docker.image.build receives a path of folder which contains Dockerfile, so i chose to create a folder for each stage dockerfile.